### ISSUE TITLE
Endpoint /me returns a non-standard error

### ISSUE THREAD

**Issue by jankovicgd** [Open-EO/openeo-openshift-driver#22]
Sending an non-authenticated request to the endpoint `/me` yields a `500 Internal server error`. In this case it should be `401 Authorization required `

**Comment by jankovicgd** [comment]
Currently this will capture a non authorized error as 400 and it will also handle any 500 errors. Maybe see how this can be a 401 and still capture 500 without resorting to use `APIException`

**Comment by jankovicgd** [comment]
Also some other endpoints that have expired tokens report something like 

```
{
  "code": 500,
  "id": "62923a7b-02cc-44c6-8293-529a16fa601f",
  "links": [
    "http://localhost:3000/redoc"
  ],
  "message": "401: b'{\"error\":\"invalid_request\",\"error_description\":\"User session not found or doesn\\'t have client attached on it\"}'",
  "service": null
}
```

**Comment by jankovicgd** [comment]
Some more fixes in #45
